source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -380,7 +380,12 @@ public TService CreateClient<TService>(Action<ServiceEndPoint> modifyServiceEndp
             public void Dispose()
             {
                 var logger = new SerilogLoggerBuilder().Build().ForContext<ClientAndService>();
-                logger.Information("Dispose called");
+                
+                logger.Information("****** ****** ****** ****** ****** ****** ******");
```

Comment:
Trying to make the logs easier to read. We get lots of errors after Dispose as everything is shutting down. Ideally, we would suppress those, but that is more involved. This is a quick bandaid so visually we can see when we dispose.